pages/partials/submissionStatus.ejs

```diff
@@ -6,13 +6,13 @@
             <span class="badge badge-danger">invalid, not gradable</span>
         <% } %>
     <% } else { %>
-        <% if (submission.grading_method == 'External') { %>
+        <% if (submission.grading_method_external) { %>
             <% if (submission.gradable) { %>
                 <span id="grading-status-<%= submission.id %>" class="badge badge-secondary"></span>
             <% } else { %>
                 <span class="badge badge-danger">invalid, not gradable</span>
             <% } %>
-        <% } else if (submission.grading_method == 'Manual') { %>
+        <% } else if (submission.grading_method_manual) { %>
```

Comment:
This may require some inspection to ensure that it works with the new logic... not just the boolean changes, but the new manual grading approach in general.

Reply:
I know this works with the workflow from before--I am going to do a sanity check on most of these pages to ensure backwards compatibility though.